doc/api/process.md

```diff
@@ -1667,6 +1680,32 @@ process.stdin.pipe(process.stdout);
 *Note*: `process.stdout` differs from other Node.js streams in important ways,
 see [note on process I/O][] for more information.
 
+## process.traceDeprecation
```

Comment:
It seems this should be placed after the `process.title`, ABC-wise.

Reply:
my bad -- don't know why I thought a ### about IO was a ## 😞